src/python/python/xml.py

```diff
@@ -0,0 +1,617 @@
+import numpy as np
+import os
+from shutil import copy2
+import math
+
+class Files:
+    MAIN = 0
+    MATS = 1
+    GEOM = 2
+    EMIT = 3
+    CAMS = 4
+    #TODO: Volumes
+
```

Comment:
This is a big file, it would be great to add some comments so futur developers can make sense out of it easily.

Reply:
I am thinking of a general description of what classes and functions in this file can be used for